components/touchableChatTile.jsx

```diff
@@ -0,0 +1,68 @@
+import {
+  View,
+  Text,
+  Image,
+  StyleSheet,
+  TouchableHighlight,
+  TouchableNativeFeedback,
+} from "react-native";
+
+export default function touchableChatTile() {
+  function _onPressButton() {
+    console.log("Pressed tile");
+  }
+
+  return (
+    <TouchableHighlight onPress={_onPressButton} underlayColor={"white"}>
+      <View
```

Comment:
why did you not use the already made component of chatTile and copy pasted instead

 <TouchableHighlight onPress={_onPressButton} underlayColor={"white"}>
<ChatTile />
 <TouchableHighlight />